etc/swtpm_setup.conf

```diff
@@ -0,0 +1,2 @@
+create_certs_tool=swtpm_localca
+create_certs_tool_config=etc/swtpm-localca.conf
```

Comment:
do we need create_certs_tool_options or this is optional?

Reply:
It is necessary. `etc/swtpm-localca.conf` contains paths to files that must be relocated to the workspace.

Reply:
You mentioned `create_certs_tool_options`. I confirmed it works without the parameter.